[2025-02-19T17:18:47.550+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T17:18:47.610+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-19T17:18:47.640+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-19T17:18:47.653+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-19T17:18:47.690+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): meteo_operations.collect_meteo_data> on 2025-02-18 00:00:00+00:00
[2025-02-19T17:18:47.713+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_data_pipeline', 'meteo_operations.collect_meteo_data', 'scheduled__2025-02-18T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/operators/testdag.py', '--cfg-path', '/tmp/tmpyuafkic0']
[2025-02-19T17:18:47.740+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask meteo_operations.collect_meteo_data
[2025-02-19T17:18:47.747+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1413) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-19T17:18:47.750+0000] {standard_task_runner.py:72} INFO - Started process 1431 to run task
[2025-02-19T17:18:47.876+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_data_pipeline.meteo_operations.collect_meteo_data scheduled__2025-02-18T00:00:00+00:00 [running]> on host b463689998de
[2025-02-19T17:18:48.169+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_data_pipeline' AIRFLOW_CTX_TASK_ID='meteo_operations.collect_meteo_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T00:00:00+00:00'
[2025-02-19T17:18:48.173+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T17:18:49.665+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': 21.4901, 'longitude': 39.1862, 'timestamp': '2025-02-19 00:00:00', 'temperature': 21.473501205444336, 'humidity': 70.0, 'wind_speed': 11.17740535736084, 'pm2_5': 71.69999694824219, 'co': 170.0}
[2025-02-19T17:18:49.715+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T17:18:49.717+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=weather_data_pipeline, task_id=meteo_operations.collect_meteo_data, run_id=scheduled__2025-02-18T00:00:00+00:00, execution_date=20250218T000000, start_date=20250219T171847, end_date=20250219T171849
[2025-02-19T17:18:49.820+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T17:18:49.885+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-19T17:18:49.895+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
